articles/linux-disk-encryption-end-user.md

```diff
@@ -0,0 +1,51 @@
+# Encrypting your Fleet-managed Linux device
+
+LUKS (Linux Unified Key Setup) is a standard tool for encrypting Linux disks. It uses a "volume key" to encrypt your data, and this key is protected by passphrases. LUKS supports multiple passphrases, allowing you to securely share access or recover encrypted data. Fleet uses LUKS to ensure that only authorized users can access the data on your work computer. 
+
+Fleet securely stores a recovery key to ensure that the data on your work computer is always recoverable. To get your computer set up for key escrow, you will first need to enable disk encryption on your end, then provide your encryption passphrase to Fleet.
+
+Follow the steps below to get set up.
+
+
+## 1. **Enable encryption during installation**
+
+  ### Ubuntu Linux
+
+  - When installing Ubuntu, choose the option to "Use LVM with encryption."
+  - Set a strong passphrase when prompted. This passphrase will be used to encrypt your disk and is separate from your login password.
+
+  <!-- TODO: screenshot of Ubuntu setup -->
```

Comment:
i'll grab this and the Fedora screenshot

Reply:
What if I already installed Ubuntu before reading this guide? How do I see if I have that enabled, and enable it if I don't?